.github/pull_request_template.md

```diff
@@ -1,10 +1,10 @@
-### Description
-
+### Checklist
 
+- [ ] I added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users.
+- [ ] I linked any relevant issues from https://github.com/maplibre/maplibre-navigation-ios/issues
```

Comment:
I think this is what was meant by the previous "Open Tasks" heading, but feel free to correct me if I'm wrong here. I wasn't really sure what it meant.